pkgs/top-level/aliases.nix

```diff
@@ -643,6 +643,7 @@ mapAliases {
   kramdown-rfc2629 = throw "'kramdown-rfc2629' has been renamed to/replaced by 'rubyPackages.kramdown-rfc2629'"; # Converted to throw 2024-10-17
   krb5Full = krb5;
   krita-beta = throw "'krita-beta' has been renamed to/replaced by 'krita'"; # Converted to throw 2024-10-17
+  ktop = kubectl-ktop throw "'ktop' has been removed as duplicate to 'kubectl-ktop'";
```

Comment:
Is the combination of throw and having an alias actually possible or wouldn't it be more a `lib.warnOnInstantiate`?

Reply:
Thanks, updated.